deploy/helm/pulumi-operator/values.yaml

```diff
@@ -4,6 +4,9 @@ deploymentStrategy: ""
 # -- Specifies the replica count for the deployment
 replicaCount: 1
 
+# -- Specifies termination grace period
+terminationGracePeriodSeconds:
```

Comment:
Can you enter a meaningful default value

Reply:
Done.